CaseStudy.py

```diff
@@ -9,8 +9,21 @@
 
 import numpy as np 
 
-np.random.seed(123) #initializes random numbers
+np.random.seed(123) # initializes random numbers
+random_walk = [0] # initializes the random_walk

```

Comment:
This is a list that will store the subsequent steps, thus creating the random walk